comp426-backend/data/account.json

```diff
@@ -14,6 +14,13 @@
       "com": {
         "passwordHash": "$2b$10$HYx72aKE87ey/KQyFHdAseGCcFSdJuyw1zSbKg5nyE75dEcdIUku6"
       }
+    },
+    "chris": {
+      "passwordHash": "$2b$10$dfl54eRYTwU0bYSbfGXNXe./FaQHsCL4K0MCwdmHyck0PDpNq.D02",
+      "data": {
+        "role": 2,
+        "description": "Lazy..."
+      }
```

Comment:
what's this for?

Reply:
we can still merge lol just wondering

Reply:
Debug/test for backend. Will delete once it's functioning. Need to get UI first.